### PR TITLE
Add new azure-ai-inference lib to java-packages.csv

### PR DESCRIPTION
**Pull request by glharper**

### CODE REVIEW

**Review by dargilco — APPROVED** [review]

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 "Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
-"azure-ai-inference","com.azure","","1.0.0-beta.1","Azure AI Inference","AI","ai","","","client","true","","","","","","","","","","","
+"azure-ai-inference","com.azure","","1.0.0-beta.1","Azure AI Inference","AI","ai","","","client","true","","","","","","","","","","",""
```
Comment (ronniegeraghty) [inline _data/releases/latest/java-packages.csv:2]:
```suggestion
"azure-ai-inference","com.azure","","1.0.0-beta.1","Azure AI Inference","Cognitive Services","ai","","","client","true","","","","","","","","","","",""
```

**Review by dargilco — DISMISSED** [review]

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (ronniegeraghty) [reply _data/releases/latest/java-packages.csv:2]: All AI services fall under the `Cognitive Services` service name.

**Review by ronniegeraghty — CHANGES_REQUESTED** [review]

**Comment by weshaggard** [comment]
Why are you trying to add this package manually? It should get added once the package ships

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -1,4 +1,5 @@
 "Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
+"azure-ai-inference","com.azure","","1.0.0-beta.1","Azure AI Inference","Cognitive Services","ai","","","client","true","","","","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:2]: Please find the other existing entry for "azure-ai-inference" in this file and update the values there.

Comment (glharper) [reply _data/releases/latest/java-packages.csv]: @weshaggard [Updated](https://github.com/Azure/azure-sdk/pull/8023/commits/ad4abd2a1a29a3c3a41cd244969b44a2ee910b1f)

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (glharper) [reply _data/releases/latest/java-packages.csv]: @ronniegeraghty Updated.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**